sdk/azidentity/CHANGELOG.md

```diff
@@ -1,17 +1,17 @@
 # Release History
 
-## 1.6.0-beta.3 (Unreleased)
+## 1.6.0-beta.3 (2024-04-09)
```

Comment:
Should this be `1.6.0` like in `BREAKING_CHANGES.md` or should the above be `1.6.0-beta.3`?

Reply:
I'm publishing beta.3 but I listed the stable version in the changes doc to avoid implying this is relevant only when upgrading to a beta version. I can hold off on adding `BREAKING_CHANGES.md` until the v1.6.0 release.